File: C++/Estrutura condicional/Dardo.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <climits>
#include <cmath>

using namespace std;

int main()
{

    double distancia, maiorDistancia, a, b, c;

    cout << "Digite as tres distancias: " << endl;
    cin >> a;
    cin >> b;
    cin >> c;

    if (a > b && a > c)
    {
        maiorDistancia = a;
    }
    else if (b > c)
    {
        maiorDistancia = b;
    }
    else
    {
        maiorDistancia = c;
    }

    cout << fixed << setprecision(2);
    cout << "MAIOR DISTANCIA = " << maiorDistancia << endl;

    return 0;
}

File: Python/Estrutura Condicional/Dardo.py
a: float; b: float; c: float; maior: float

print(f"Digite as 3 distancias:")
a = float(input())
b = float(input())
c = float(input())

if (a > b) and (a > c):
    maior = a
elif (b > a) and (b > c):
    maior = b
else:
    maior = c


print(f"MAIOR DISTANCIA = {maior:.2f}")